app/models/driver.rb

```diff
@@ -0,0 +1,52 @@
+class Driver < ApplicationRecord
+  has_many :trips
+
+  validates :name, presence: true
+  validates :vin, presence: true, length: { is: 17 }
+  validates :available, inclusion: { in: ["true", "false"] }
```

Comment:
If this was a boolean (per the earlier note) then this could just be:

```suggestion
  validates :available, presence: true
```